internal/testgen/main.go

```diff
@@ -26,6 +26,7 @@ var freeTestsGenerators = map[string]TestsGenerator{ // by subdirectory.
 var checkerTestsGenerators = []CheckerTestsGenerator{
 	BlankImportTestsGenerator{},
 	BoolCompareTestsGenerator{},
+	ContainsTestsGenerator{},
```

Comment:
Maybe move this one after compare, it might find use case that needs to be simplified before contains can be used